hera_librarian/cli.py

```diff
@@ -187,14 +190,25 @@ def config_add_file_event_subparser(sub_parsers):
 
     # add sub parser
     sp = sub_parsers.add_parser("add-file-event", description=doc, help=hlp)
-    sp.add_argument("conn_name", metavar="CONNECTION-NAME", type=str,
-                    help=_conn_name_help)
-    sp.add_argument("file_path", metavar="PATH/TO/FILE", type=str,
-                    help="The path to file in librarian.")
-    sp.add_argument("event_type", metavar="EVENT-TYPE", type=str,
-                    help="The type of event.")
-    sp.add_argument("key_vals", metavar="key1=val1...", type=str, nargs="+",
-                    help="key-value pairs of events.")
+    sp.add_argument(
+        "conn_name", metavar="CONNECTION-NAME", type=str, help=_conn_name_help
+    )
+    sp.add_argument(
+        "file_path",
+        metavar="PATH/TO/FILE",
+        type=str,
+        help="The path to file in librarian.",
+    )
+    sp.add_argument(
+        "event_type", metavar="EVENT-TYPE", type=str, help="The type of event."
+    )
+    sp.add_argument(
+        "key_vals",
+        metavar="key1=val1...",
+        type=str,
+        nargs="+",
+        help="key-value pairs of events.",
+    )
     sp.set_defaults(func=add_file_event)
```

Comment:
After writing this submodule, it has since struck me as odd that I decided to put the target function definitions after they are specified here. (I tend to write code so that functions are defined before they are used, though obviously this is not strictly required.) Refactoring so the functions (like `add_file_event`) come before the subparser definitions has never been a high priority, but now that we're significantly editing this file, I think I'd prefer to move the subparser definitions to be below the target function definitions.